config.js

```diff
@@ -1,17 +1,17 @@
 try {
-  require('dotenv').config();
-} catch (e) { 
-  console.error('error loading dotenv', e);
+  require("dotenv").config();
+} catch (e) {
+  console.error("error loading dotenv", e);
 }
 
 module.exports = {
   twilio: {
-    accountSid: process.env.TWILIO_ACCOUNT_SID,
-    authToken: process.env.TWILIO_AUTH_TOKEN,
-    apiKey: process.env.TWILIO_API_KEY,
-    apiSecret: process.env.TWILIO_API_SECRET,
-    chatServiceSid: process.env.TWILIO_CHAT_SERVICE_SID
+    accountSid: "AC58941c4201fa09443ae7464d5f4d49aa",
```

Comment:
@azamoras  -- Please clone the repo and update the config file with your own credentials. You don't want to expose sensitive material like this on a Github PR.

Please reach out to Twilio Support to cycle your credentials.